File: WorldConquest/Models/Country.cs
namespace WorldConquest.Models
{
    public class Country
    {
        // Unique country Id
        public string Id { get; set; }

        // Display name of the country
        public string Name { get; set; }

        // Color used for provinces (e.g., "R_G_B" or hex)
        public string Color { get; set; }

        // Money owned by the country
        public int Money { get; set; }

        // Player Id controlling this country (null if AI or unassigned)
        public string? PlayerId { get; set; }

        // Additional metrics (expand as needed)
        public int Stability { get; set; }
        public int Population { get; set; }
    }
}

File: WorldConquest/client/src/models/Country.ts
export interface Country {
  // Unique country Id
  id: string;

  // Display name of the country
  name: string;

  // Color used for provinces (e.g., "R_G_B" or hex)
  color: string;

  // Money owned by the country
  money: number;

  // Player Id controlling this country (null if AI or unassigned)
  playerId: string | null;

  // Additional metrics (expand as needed)
  stability: number;
  population: number;
}